src/components/timeline/Timeline.svelte

```diff
@@ -43,6 +46,14 @@
     return { date, time, yearDay };
   });
 
+  onMount(() => {
+    document.addEventListener('keydown', onKeyDown);
```

Comment:
Instead of the `onMount` event listener bindings can we add:

```svelte
<svelte:window on:keydown={onKeydown} />
```

This should automatically add/remove the key listener from this component.

Reply:
Oh nice I didn't know about that, that's much cleaner.